### PR TITLE
Fix facility feeback health when 0 values

### PR DESCRIPTION
**Pull request by brangi**
## Description

issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/26588

### CODE REVIEW

File: src/applications/static-pages/tests/facilities/FacilityPatientSatisfactionScoresWidget.unit.spec.jsx [hunk 1]
```diff
@@ -51,4 +54,19 @@ describe('facilities <FacilityPatientSatisfactionScoresWidget>', () => {
...
+
+  it('should NOT render facility patient satisfaction scores section when 0 values', () => {
+    const tree = shallow(
+      <FacilityPatientSatisfactionScoresWidget
+        loading={false}
+        facility={mockFeedbackZeroValues.data[0]}
+      />,
+    );
+    expect(tree).to.be.empty;
+    expect(tree.find('LoadingIndicator').exists()).to.be.false;
```
Comment (mpelzsherman) [inline src/applications/static-pages/tests/facilities/FacilityPatientSatisfactionScoresWidget.unit.spec.jsx:66]: Can we use `not.to.exist` here instead?

File: src/applications/static-pages/tests/facilities/mockFacilitiesData.js [hunk 2]
```diff
@@ -81,4 +81,91 @@ const mockFacilityLocatorApiResponse = {
...
+            address2: null,
+            address3: 'Suite 201',
+          },
+        },
+        classification: 'Primary Care CBOC',
+        detailedServices: null,
+        facilityType: 'va_health_facility',
+        feedback: {
+          health: {
+            primaryCareUrgent: 0.0,
```
Comment (mpelzsherman) [inline src/applications/static-pages/tests/facilities/mockFacilitiesData.js:122]: Maybe change one of these to non-zero just to cover that case?

Comment (brangi) [reply src/applications/static-pages/tests/facilities/mockFacilitiesData.js:122]: there is already this one https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/static-pages/tests/facilities/FacilityPatientSatisfactionScoresWidget.unit.spec.jsx#L7 , need one more?

File: src/applications/static-pages/tests/facilities/FacilityPatientSatisfactionScoresWidget.unit.spec.jsx (hunk 1 above)
Comment (brangi) [reply src/applications/static-pages/tests/facilities/FacilityPatientSatisfactionScoresWidget.unit.spec.jsx:66]: So I changed `expect(tree.find('FacilityPatientSatisfactionScoresWidget').exists()).to.be` to `expect(tree.find('FacilityPatientSatisfactionScoresWidget')).not.to.exist;` But it doesn't pass

Comment (brangi) [reply src/applications/static-pages/tests/facilities/FacilityPatientSatisfactionScoresWidget.unit.spec.jsx:66]:
``` 1) facilities <FacilityPatientSatisfactionScoresWidget>
       should NOT render facility patient satisfaction scores section when 0 values:
     AssertionError: expected {} to not exist
      at Context.<anonymous> (src/applications/static-pages/tests/facilities/FacilityPatientSatisfactionScoresWidget.unit.spec.jsx:67:5)
      at Test.Runnable.run (node_modules/mocha-snapshots/src/index.js:19:22)
      at processImmediate (internal/timers.js:461:21)
```

Comment (mpelzsherman) [reply src/applications/static-pages/tests/facilities/FacilityPatientSatisfactionScoresWidget.unit.spec.jsx:66]: OK, np, thanks. We can leave as is.

**Review by mpelzsherman — APPROVED** [review]

**Pull request merged**